bigbluebutton-html5/imports/ui/components/app/clearSession.js

```diff
@@ -0,0 +1,13 @@
+import Storage from '/imports/ui/services/storage';
+
+export function clearSessionVar(callback) {
```

Comment:
This problably should be moved to the Auth service `/imports/ui/services/auth/index.js` and renamed to `clearCredentials`. The idea is to have all Auth behaviors and methods on the same service.